src/main/java/com/marginallyclever/makelangelo/Makelangelo.java

```diff
@@ -658,7 +652,7 @@ private JMenu createViewMenu() {
 		menu.add(buttonZoomToFit);
 
 		JCheckBoxMenuItem checkboxShowPenUpMoves = new JCheckBoxMenuItem(Translator.get("GFXPreferences.showPenUp"),GFXPreferences.getShowPenUp());
-		checkboxShowPenUpMoves.setAccelerator(KeyStroke.getKeyStroke("shift M"));
```

Comment:
imo shift should never been an accelerator key.  I'm thinking of "your message here" where you can type in any language and that will eventually have someone type shift+letter to start a sentence.